docs/migratetoxcpng.md.orig

```diff
@@ -0,0 +1,128 @@
+# Migrate to XCP-ng
+
+If you are using another virtualization platform (VMware, KVM, etc.), this part of the documentation will help you to migrate to XCP-ng.
+
+:::warning
+OVA import will miss the information if the VM is running BIOS or UEFI mode. Double check your settings on your original system, and then enable (or not) UEFI on XCP-ng side for the destination VM. You can do so in VM advanced tab in Xen Orchestra.
+:::
+
+## From XenServer
+
+We got a dedicated section on [how to migrate from XenServer to XCP-ng](upgrade.md#upgrade-from-xenserver).
+
+## From Citrix Hypervisor
+
+We got a dedicated section on [how to migrate from Citrix Hypervisor to XCP-ng](upgrade.md#upgrade-from-xenserver).
+
+<<<<<<< HEAD
+## From Xen on Linux
+
+If you are running Xen on your usual distro (Debian, Ubuntu…), you are using `xl` to manage your VMs, and also plain text configuration files. You can migrate to an existing XCP-ng host thanks to [this Python script](http://www-archive.xenproject.org/files/xva/xva.py).
+
+1. Get that script in your current `dom0`.
+1. Shutdown your VM
+1. Run the script, VM by VM with for example: `./xva.py -c /etc/xen/vm1.cfg -n vm1 -s xcp_host_1 --username=root --password="mypassword" --no-ssl`. You can use a hostname or the IP address of your XCP-ng host (name `xcp_host_1` here)
+1. Your disks are streamed while the configuration file is "translated" to a VM object in your XCP-ng host.
+1. As soon it's done, you should be able to boot your VM on destination
+1. Repeat for all your VMs
+
+If you have an error telling you that you don't have an default SR, please choose a default SR on your XCP-ng pool (in XO, Home/Storage, hover on the storage you want to put by default, there's an icon for it).
+
+:::warning
+This script is a bit old and not tested since while. If you have issues, feel free to report that!
+:::
+
+## From Virtualbox
+=======
+## From VirtualBox
+>>>>>>> 2ad9372 (docs/migratetoxcpng.md: style cleanup)
```

Comment:
Bad rebase? :sweat_smile:

Reply:
Lol yeah, let me change this to a WIP because I have that and a few other bad artifacts fixed locally.  Just want to double check it is all cleaned up before pushing v2.